manifests/service.pp

```diff
@@ -13,7 +13,7 @@
     mode    => '0755',
   }
 
-  if $confluence::manage_service {
+  if str2bool($confluence::manage_service) {
     service { 'confluence':
```

Comment:
According to [this documentation](https://docs.puppet.com/puppet/4.10/lang_updating_manifests.html#empty-strings-in-boolean-context-are-true) , Puppet 4 treats an empty string as True. The opposite was the case in < 4 so str2bool is now required, if one is being paranoid, in the event that a user sends in an empty string instead of a boolean value. I guess a more proper Puppet 4 DSL would declare `Boolean $manage_service` in `init.pp`. LMK if you want me to do this.